cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlClientTest.java

```diff
@@ -215,6 +215,70 @@ public void testCCRemoveBroker(Vertx vertx, VertxTestContext context) throws IOE
                 });
     }
 
+    @Test
+    public void testCCRemoveBrokerDisks(Vertx vertx, VertxTestContext context) throws IOException, URISyntaxException {
+        BrokerAndVolumeIds brokerAndVolumeIds = new BrokerAndVolumeIdsBuilder()
+                .withVolumeIds(1, 2, 3)
+                .withBrokerId(0)
```

Comment:
just a nit, but I would use `withBrokerId` before `withVolumeIds` so it follows the kind of logic ... broker down to volumes.